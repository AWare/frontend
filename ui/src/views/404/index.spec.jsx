// @flow

import test from 'utils/test-render';
import fourOhFour from '.';

describe('404', () => {
    test(<fourOhFour />);
});
