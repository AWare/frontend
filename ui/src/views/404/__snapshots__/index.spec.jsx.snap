// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 generates the expected markup 1`] = `
Object {
  "css": "<style class=\\"_styletron_hydrate_\\"></style>",
  "html": "<fourOhFour></fourOhFour>",
}
`;
